site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -192,6 +192,21 @@ CustomTabsIntent intent = new CustomTabsIntent.Builder()
     .setSecondaryToolbarViews(secondaryToolbarViews, clickableIds, toolbarPendingIntent)
     .build();
 ```
+## Add an action button to the toolbar
```

Comment:
Would it make sense to link to this section from the [existing toolbar button section](https://developer.chrome.com/docs/android/custom-tabs/guide-interactivity/#add-a-custom-action-button)?

Reply:
I could even move them closer together and then cross-link them.